public/style.css

```diff
@@ -586,7 +586,10 @@ a img {
 #import,
 #settings,
 #printkeymaps,
+#testkeys,
 #leavePrint,
+#leaveTest,
+#resetTest,
```

Comment:
why using ID target and not just a class for this?

Reply:
This is mostly a legacy issue, the style sheet was done with IDs instead of classes, so it's inertia. We're going to redesign the UI, so I'm not too concerned, but it's definitely an anti-pattern these days.